Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.yml

```diff
@@ -1,10 +1,10 @@
 commonfields:
   id: PerformActionOnCampaignIncidents
   version: -1
-dockerimage: demisto/python3:3.9.5.20070
+dockerimage: demisto/python3:3.9.6.22912
 enabled: true
 name: PerformActionOnCampaignIncidents
-runas: DBotWeakRole
+runas: DBotRole
```

Comment:
can you elaborate why? (add to rn as well)

Reply:
Since we are using the delete context and set the incident context, this requires elevated permissions according to validate.
Will add to RN.